include/swift/AST/Decl.h

```diff
@@ -750,7 +750,11 @@ class alignas(1 << DeclAlignInBits) Decl : public ASTAllocated<Decl> {
     ObjCNameLookupCachePopulated : 1,
 
     /// Whether this module has been built with C++ interoperability enabled.
-    HasCxxInteroperability : 1
+    HasCxxInteroperability : 1,
+
+    /// Whether this module has been built with -experimental-skip-non-exportable-decls
+    /// or -experimental-skip-non-inlinable-function-bodies.
```

Comment:
It looks like the logic sets this bit only for `-experimental-skip-non-exportable-decls`. Should we remove the mention of `-experimental-skip-non-inlinable-function-bodies ` from the comment? I believe the optimization should work fine with just `-experimental-skip-non-inlinable-function-bodies`.